backend/api/api/views.py

```diff
@@ -0,0 +1,36 @@
+from django.http import HttpResponse
+
+#importing functions from folder
+import backend.functions as functions
+from functions.register import register
+from functions.login import login
+
+current_user = ''
+
+def register_user(request):
+    result = 1
+    if request.method == 'POST':
+        username = request.POST['username']
+        password = request.POST['password']
+        result = register(username, password)
+    if result == 0:
+        pass #should redirect to new page
+    elif result == 2:
+        pass #create message that username already exists
+    else:
+        pass #username and password too short or long
+
+def login_user(request):
```

Comment:
I think a user logging in should be a GET request. We should just check if the username and password are a match, but there should be no changes to the database.

Reply:
I'm not familiar with HTTP request methods, but the retrieving of the data happens inside the functions that the API calls with SQLalchemy and not with the built-in Django functions. I think the method should still be POST because the front-end is sending data from the login form, but I'm not really sure.